### PR TITLE
Cleanup from fixed softfailure

### PR DESCRIPTION
**Pull request by asmorodskyi**

### CODE REVIEW

**Comment by asmorodskyi** [comment]
not sure if I need proof run here . as I just removing dead code basically

**Comment by asmorodskyi** [comment]
which needles do you think related to this PR ?

File: tests/installation/accept_license.pm [hunk 1]
```diff
@@ -35,16 +35,7 @@ sub run {
     push @tags, 'inst-welcome-no-product-list' if is_sle('15+') && get_var('UPGRADE');
 
     assert_screen \@tags;
-    if (match_has_tag('network-settings-button')) {
```
Comment (rwx788) [inline tests/installation/accept_license.pm]: The ones used in removed piece of code, namely with network-settings-button tag. These are the only usages, so also, please, clean-up  tags array on the line 27

File: tests/installation/accept_license.pm [hunk 2]
```diff
@@ -35,16 +35,7 @@ sub run {
...
-        # workaround for hpc missing license: https://bugzilla.suse.com/show_bug.cgi?id=1060174
-        if (check_var('SLE_PRODUCT', 'hpc')) {
-            record_soft_failure('bsc#1060174');
-            return;
-        }
-        else {
-            die 'It seems that license agreement is missing, please check!';
-        }
-    }
+    die 'It seems that license agreement is missing, please check!' if match_has_tag('network-settings-button');
```
Comment (asmorodskyi) [inline tests/installation/accept_license.pm]: the tag is still in use here

File: tests/installation/accept_license.pm (hunk 1 above)
Comment (asmorodskyi) [reply tests/installation/accept_license.pm]: tag comment below ...

File: tests/installation/accept_license.pm (hunk 2 above)
Comment (rwx788) [reply tests/installation/accept_license.pm]: Ah, my bad, missed this point. Could you please then undo line #64 removal and I'll merge. Thanks!

Comment (asmorodskyi) [reply tests/installation/accept_license.pm]: done

**Pull request merged**